Compilation error

```
ERROR in src/app/dev/fakeBackEnd.ts:12:7 - error TS2451: Cannot redeclare block-scoped variable 'PROJECT'.

12 const PROJECT: ProjectData = {
         ~~~~~~~
src/app/dev/fakeBackEnd.ts:63:7 - error TS2451: Cannot redeclare block-scoped variable 'PROJECT'.

63 const PROJECT: ProjectData = {
         ~~~~~~~

** Angular Live Development Server is listening on localhost:4200, open your browser on http://localhost:4200/ **
```

--- a/src/app/dev/fakeBackEnd.ts
+++ b/src/app/dev/fakeBackEnd.ts
@@ -1,4 +1,4 @@
-import { Injectable, isDevMode } from '@angular/core';
+import { Injectable } from '@angular/core';
 import { HttpRequest, HttpResponse, HttpHandler, HttpEvent, HttpInterceptor, HTTP_INTERCEPTORS } from '@angular/common/http';
 import { Observable, of, throwError } from 'rxjs';
 import { delay, mergeMap, materialize, dematerialize } from 'rxjs/operators';
@@ -55,36 +55,6 @@ const users: UserAccountData[] = [
     },
 
 ];
-
-
-
-
-
-const PROJECT: ProjectData = {
-    id: 0,
-    name: "Projet Vichy",
-    description: "A collaboration project",
-    member_ids: [0],
-    permissions: [
-        {
-            member_id: 0,
-            permissions: {
-                accessible_files: true,
-                create_post: true,
-                delete_file: true,
-                deposit_file: true,
-                manage_members: true,
-                manage_permission: true,
-                manage_project: true
-            }
-        },
-    ],
-    file_ids: [],
-    links: [],
-    post_ids: [0],
-    visibility: true,
-}
-
 
 @Injectable()
 export class FakeBackendInterceptor implements HttpInterceptor {

--- a/src/app/models/classes/PostBase.ts
+++ b/src/app/models/classes/PostBase.ts
@@ -13,6 +13,7 @@ interface MergeablePostBaseData {
 interface PostBase {
     readonly liked: boolean;
     readonly content: string;
+    getLikers(): Promise<UserAccount[]>;
     edit(content: string): Promise<this>;
     like(): Promise<this>;
     unlike(): Promise<this>;
@@ -55,6 +56,11 @@ export abstract class PostBaseObject<Data extends PostBaseData> extends Mergeabl
     get liked() {
         const userId = this.api.user?.id;
         return userId ? this._likes.includes(userId) : false;
+    }
+
+    async getLikers() {
+        await this.fetch();
+        return await Promise.all(this._likes.map(id => this.api.users.get(id)));
     }
 
     async edit(content: string) {

--- a/src/app/models/classes/UserAccount.ts
+++ b/src/app/models/classes/UserAccount.ts
@@ -71,7 +71,10 @@ export class UserAccount extends MergeableApiObject<MergeableUserAccountData, Us
 
     get links() { return this._links as ReadonlyArray<string>; }
 
-    get projectIds() { return this._projectIds as ReadonlyArray<number>; }
-
     get profilePictureUrl() { return `/api/profilePicture/${this.id}`; }
+
+    async getProjects() {
+        await this.fetch();
+        return await Promise.all(this._projectIds.map(id => this.api.projects.get(id)));
+    }
 }

